multiUrlGenerator.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
This file should not be commited, since this is the transpiled ts code that we already have. Please delete this file and push the deletion.